app/components/black-lives-matter-banner.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if this.show}}
+  <div class="survey-reminder-wrapper">
+    {{#link-to 'community.black-lives-matter'}}Black Lives Matter{{/link-to}}
```

Comment:
I think we can use the angle bracket syntax in this project now.

```suggestion
    <LinkTo @route="community.black-lives-matter">Black Lives Matter</LinkTo>
```

Reply:
👍